x-pack/plugins/apm/server/lib/services/get_service_transaction_groups/merge_transaction_group_data.ts

```diff
@@ -23,12 +16,14 @@ export function mergeTransactionGroupData({
   transactionGroups,
   timeseriesData,
   latencyAggregationType,
+  transactionType,
 }: {
   start: number;
   end: number;
   transactionGroups: TransactionGroupWithoutTimeseriesData[];
   timeseriesData: TransactionGroupTimeseriesData;
   latencyAggregationType: LatencyAggregationType;
+  transactionType: string;
```

Comment:
I think there's still a terms aggregation on transaction.type in the original search that we might not need anymore. Can you have a look? (Might be that we are doing it in other places that can now be removed as well).

Reply:
Yeah, I'm going to remove this aggregation https://github.com/cauemarcondes/kibana/blob/apm-duplicate-api-calls/x-pack/plugins/apm/server/lib/services/get_service_transaction_groups/get_timeseries_data_for_transaction_groups.ts#L90-L94